Update furnace and boiler efficiency to match values from ASHRAE 90.1-2004 through 2019

jugonzal07 opened the pull request:
The proposed changes includes two changes: 
1) Update 90.1-2019 boilers to use the correct 84% minimum annual fuel utilization efficiency in high capacity boilers
2) Update the 90.1 2004-2019 furnace gas heating coils to use their respective minimum efficiency values (based on year and capacity), updated with relevant documentation. This required adding json files for furnaces to the repository and workflow.

For more information regarding the motivation for this update, consider looking into [Addendum 90.1-2019 BP and Addendum 90.1-2019 BO](https://www.ashrae.org/file%20library/technical%20resources/standards%20and%20guidelines/standards%20addenda/90.1-2016/90_1_2016_be_bm_bn_bo_bp_br_bs_bu_bv_cf_cl_cm_cq_ct_cu_cv_cw_cy_20210324.pdf).

mdahlhausen reviewed (changes requested):
This commit is almost ready. I added thermal efficiency data for the doe ref and AEDG standards.  Please rebuild the furnaces.json files (2 ref buildings, 2 AEDG jsons that will be new).

jugonzal07 commented:
> This commit is almost ready. I added thermal efficiency data for the doe ref and AEDG standards. Please rebuild the furnaces.json files (2 ref buildings, 2 AEDG jsons that will be new).

Hi Matt. I updated the JSONs with the changes you made to the standards spreadsheet.
